traitsui/tests/editors/test_range_editor.py

```diff
@@ -113,16 +159,122 @@ def check_set_with_text_after_empty(self, mode):
             self.assertEqual(displayed, str(model.value))
 
     def test_simple_slider_editor_set_with_text_after_empty(self):
-        return self.check_set_with_text_after_empty(mode='slider')
+        return self.check_slider_set_with_text_after_empty(mode='slider')
 
     def test_large_range_slider_editor_set_with_text_after_empty(self):
-        return self.check_set_with_text_after_empty(mode='xslider')
+        return self.check_slider_set_with_text_after_empty(mode='xslider')
 
     def test_log_range_slider_editor_set_with_text_after_empty(self):
-        return self.check_set_with_text_after_empty(mode='logslider')
+        return self.check_slider_set_with_text_after_empty(mode='logslider')
 
     # on wx the text style editor gives an error whenever the textbox
     # is empty, even if enter has not been pressed.
     @requires_toolkit([ToolkitName.qt])
     def test_range_text_editor_set_with_text_after_empty(self):
-        return self.check_set_with_text_after_empty(mode='text')
+        model = RangeModel()
+        view = View(
+            Item(
+                "value",
+                editor=RangeEditor(low=1, high=12, mode="text")
+            )
+        )
+        tester = UITester()
+        with tester.create_ui(model, dict(view=view)) as ui:
+            number_field_text = tester.find_by_name(ui, "value")
+            # Delete all contents of textbox
+            for _ in range(5):
+                number_field_text.perform(command.KeyClick("Backspace"))
+            number_field_text.perform(command.KeySequence("11"))
+            number_field_text.perform(command.KeyClick("Enter"))
+            displayed = number_field_text.inspect(query.DisplayedText())
+            self.assertEqual(model.value, 11)
+            self.assertEqual(displayed, str(model.value))
+
+    # SimpleSpinEditor not yet implemented on Wx.
```

Comment:
This comment reads to me as "the feature mode='spinner' is not implemented on Wx".
I think it would be better to say "the tester support code is not yet implemented for Wx SimpleSpinEditor".